devices.js

```diff
@@ -14001,6 +14001,20 @@ const devices = [
         description: '6W smart dimmable E27 lamp 2700K',
         extend: generic.light_onoff_brightness,
     },
+	// LoraTap SC400 curtain switch
+    {
+        zigbeeModel: ['TS130F'],
+		fingerprint: [{modelID: 'TS130F', manufacturerName: '_TZ3000_8kzqqzu4'}],
+        model: 'TS0601_curtain_switch',
+        vendor: 'TuYa',
+        description: 'Curtain/Blind switch',
+        supports: 'open, close, stop, position',
```

Comment:
Please add `exposes` here (for automatic home assistant discovery and control via frontend: https://www.zigbee2mqtt.io/information/frontend.html). 

The following should work (but please test):

```
exposes: [e.cover_position(), exposes.binary('calibration', 'rw', 'ON', 'OFF'), exposes.binary('backlight_mode', 'rw', 'ON', 'OFF'), exposes.binary('motor_reversal', 'rw', 'ON', 'OFF')],
```

Reply:
calibration and motor_reversal seems to work.
For backlight_mode i would use: 
exposes.enum('backlight_mode', 'rw', ['0', '1', '2']),